pages/docs/reference/multiplatform.md

```diff
@@ -88,8 +88,8 @@ projects is much more flexible, and we'll keep working on it to make project con
 
 ## Multiplatform Libraries
 
-Common code can rely on a set of libraries that cover everyday tasks such as HTTP, serialization and managing 
-coroutines. Also, an extensive standard library is available on all platforms. 
+Common code can rely on a set of libraries that cover everyday tasks such as [HTTP](http://ktor.io/clients/index.html), [serialization](https://github.com/Kotlin/kotlinx.serialization), and [managing 
```

Comment:
Maybe http://ktor.io/clients/http-client/multiplatform.html is a better fit?

Reply:
Indeed. I'll change the link.